scripts/d01_data/FE_load_data.py

```diff
@@ -0,0 +1,8 @@
+import pandas as pd
+import os
+
+DATA_DIR = 'data/processed'
```

Comment:
It would be neatest to move these global variables to a config file. And then load them in the script from there. That makes that for example if you change data_dir, you don't have to search in which files this is declared and change all of them.